### ISSUE TITLE
Support for `bypass_platform_safety_checks_on_user_schedule_enabled` and `reboot_setting`

### ISSUE THREAD

**Issue by davidkarlsen** [Azure/terraform-azurerm-virtual-machine#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

It would be good to control the `bypass_platform_safety_checks_on_user_schedule_enabled` and `reboot_setting` flags. We have custom RHEL boxes that we want to auto-patch.

### New or Affected Resource(s)/Data Source(s)

azurerm_linux_virtual_machine

### Potential Terraform Configuration

```hcl
var.reboot_setting = "IfRequired"
var.bypass_platform_safety_checks_on_user_schedule_enabled = true
```

### References

* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_virtual_machine#bypass_platform_safety_checks_on_user_schedule_enabled
* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_virtual_machine#reboot_setting

_No response_

**Comment by davidkarlsen** [comment]
@lonegunmanb WDYT?

**Comment by davidkarlsen** [comment]
Fixed in #66
